solr_manager/docker-compose.yml

```diff
@@ -0,0 +1,145 @@
+# a docker-compose.yml to start a simple cluster with three ZooKeeper nodes and three Solr nodes.
+services:
+  solr_manager:
+    build:
+      context: .
+      target: runtime
+      dockerfile: Dockerfile
+      args:
+        UID: ${UID:-1000}
+        GID: ${GID:-1000}
+        ENV: ${ENV:-dev}
+        POETRY_VERSION: ${POETRY_VERSION:-1.5.1}
+        SOLR_PASSWORD: ${SOLR_PASSWORD:-solr}
+        SOLR_USER: ${SOLR_USER:-solrRocks}
+        ZK_HOST: ${ZK_HOST:-zoo1:2181,zoo2:2181,zoo3:2181}
+    env_file:
+      - .env
+    volumes:
+      - .:/app
+    stdin_open: true
+    depends_on:
+      - zoo1
+      - zoo2
+      - zoo3
+      - solr1
+      - solr2
+      - solr3
+      - cluster_security
+    tty: true
+    container_name: solr_manager
+    networks:
+      - solr
+  solr1:
+    build:
+      context: ..
+      dockerfile: solr8.11.2_files/Dockerfile
+      target: embedded_zookeeper
```

Comment:
We should be using external zookeeper here?

Reply:
Also I think for our use case even with the external zookeeper target we don't need to run `solr_init.sh` (which is the default command for the external solr one too) -- since our separate `cluster_security` container takes care of that.

Reply:
You are right, we will use external zookeeper here, and I will keep the solr_init.sh to copy the security.json. I have learned how to start the cluster with Basic authentication without using the cluster_security container; then cluster_security container will be removed.